### DISCUSSION TITLE
t4240rdb(32bit) build error on Yocto scarthgap - 32bit seems to require 64bit support which is not enabled

### DISCUSSION THREAD

**Discussion by SonaSarmadi** [Freescale/meta-freescale#1893] [General]
Hi, I have been successfully built  "core-image-minimal" on Yocto scarthgap for p2020rdb (32bit SPE),  t4240rdb-64b (64bit) and now I am trying to build  t4240rdb (32bit), but I get error. I just select MACHINE ??= "t4240rdb" and add "meta-freescale" layer from scarthgap and run "bitbake core-image-minimal", I get the error below:

ERROR: Required build target 'core-image-minimal' has no buildable providers.
Missing or unbuildable dependency chain was: ['core-image-minimal', 'virtual/kernel', 'lib64-libgcc']

I want a 32bit image, (kernel & rootfs), I tried to remove multilib64, then I get other errors. 
1) Why is a 64bit toolchain needed when I build 32bit image?
2) If it is necessary of some reason, how can include/enable it?

Thanks in advance

**Comment by angolini** [comment]
I just tried on master, and the error I see is:

```
ERROR: Nothing PROVIDES 'virtual/powerpc-fsl-linux-gcc' (but /fslc/source/master/sources/meta-freescale/recipes-bsp/u-boot/u-boot-qoriq_2021.04.bb DEPENDS on or otherwise requires it)
ERROR: Required build target 'core-image-base' has no buildable providers.
Missing or unbuildable dependency chain was: ['core-image-base', 'u-boot', 'virtual/powerpc-fsl-linux-gcc']
```
@otavio, can you, please, help us making it an issue and assigning it to someone who can help?
